powersync-static/setup_powersync.c

```diff
@@ -0,0 +1,9 @@
+#include <sqlite3.h>
+
+extern int sqlite3_powersync_init(sqlite3 *db, char **pzErrMsg,
+                                  const sqlite3_api_routines *pApi);
+
+int setup_powersync()
+{
+  return sqlite3_auto_extension((void (*)(void)) & sqlite3_powersync_init);
+}
```

Comment:
You can probably skip the custom C code completely, and just use the existing `powersync_init_static` function (does exactly the same as this one).

Reply:
That is nicer. I've removed this function and file now. `powersync_init_static` is called directly in the demo and in our SDK.